server-example/requirements.txt

```diff
@@ -0,0 +1,38 @@
+appdirs==1.4.3
```

Comment:
I think that the only direct dependency that you need here is Flask - you can remove the others.